src/WebJobs.Script.WebHost/Controllers/KeysController.cs

```diff
@@ -186,11 +189,13 @@ private async Task<IActionResult> AddOrUpdateSecretAsync(string keyName, string
                 case OperationResult.Created:
                     {
                         var keyResponse = ApiModelUtility.CreateApiModel(new { name = keyName, value = operationResult.Secret }, Request);
+                        await _functionsSyncManager.TrySyncTriggersAsync();
```

Comment:
Now that keys can also be part of the sync triggers payload, we need to sync triggers whenever they are updated.